qupath-core/src/main/java/qupath/lib/objects/classes/PathClassFactory.java

```diff
@@ -148,13 +153,14 @@ static boolean classExists(String classString) {
 	 * @param name
 	 */
 	private static void validateName(String name) {
```

Comment:
I think the original implementation is 'correct' although admittedly the terms are unclear: the name component must not contain a colon, but a single `PathClass` derived from other `PathClasses` can have a `toString()` method that contains a colon as a separator.

The javadocs should probably reflect this, or rename the method - but I think the colon check should be kept.